build.toml

```diff
@@ -211,6 +222,13 @@ install_destination = 'ni-paths-LV{veristand_version}DIR\examples\NI VeriStand C
 control_file = 'control_examples'
 package_output_dir = 'Source\Built'
 
+[[package]]
+type = 'nipkg'
+payload_dir = 'Source\Shared\Hardware Custom Device API'
```

Comment:
I don't see any files at this path currently; are you expecting there to be any?

In general if you are installing VIs, `payload_dir` should be set to `Source\Built\...`, so that the VIs which are included can be properly mass-compiled, and aren't limited to the source version.